ang/civicase/ContactsDataService.js

```diff
@@ -105,5 +105,16 @@
     this.getContactIconOf = function (contactID) {
       return savedContactDetails[contactID] ? savedContactDetails[contactID].contact_type : '';
     };
+
+    /**
+     * Return current logged in user id
+     *
+     * @return {String} id of the current user
+     */
+    this.getSelectedContactId = function () {
```

Comment:
This should not be part of this service, a this service is to fetch additional data for contacts.